vms/vms/settings.py

```diff
@@ -61,8 +62,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'vms',
-        'USER': 'myuser',
-        'PASSWORD': 'mypassword',
+        'USER': 'vmsadmin',
+        'PASSWORD': '0xdeadbeef',
         'HOST': 'localhost',
     }
```

Comment:
@smarshy Please make a separate commit for this change. This is not related to the issue that you want to fix in this pull request.